File: python/armstrong_numbers.py
def is_armstrong_number(number):
    number_as_array = [int(x) for x in str(number)]
    armstrong_num = 0
    for num in number_as_array:
        armstrong_num += num**len(number_as_array)
    if armstrong_num == number:
        return True
    else:
        return False
    
def find_armstrong_numbers(numbers):
    armstrong_numbers = []
    for num in numbers:
        if is_armstrong_number(num):
            armstrong_numbers.append(num)
    return armstrong_numbers

File: js/armstrongNumbers.js
function isArmstrongNumber(num){
    numArray = Array.from(String(num), num => Number(num)); 
    armstrongNumber = 0;
    numArray.map((val, index) => {
        armstrongNumber += val**numArray.length
    })

    if (armstrongNumber == num){
        return true;
    }else{
        return false;
    }
  
}

function findArmstrongNumbers(numbers) {
    armstrongNumbers = [];
    for(let x =0;x < numbers.length;x++){
        if(isArmstrongNumber(numbers[x])){
            armstrongNumbers.push(numbers[x]);
        }
    }
    return armstrongNumbers;
}
  
module.exports = findArmstrongNumbers;
  